unittests/tools/test_solar_appscreener_parser.py

```diff
@@ -26,6 +26,8 @@ def test_solar_appscreener_parser_with_one_criticle_vuln_has_one_findings(self):
         self.assertEqual(1, len(findings))
         self.assertEqual("Hardcoded password", finding.title)
         self.assertEqual("Critical", finding.severity)
+        self.assertEqual("misc/shared.php", finding.file_path)
+        self.assertEqual("151", finding.line)
```

Comment:
This should be a number according to the data model.
```suggestion
        self.assertEqual(151, finding.line)
```